.goreleaser.yaml

```diff
@@ -1,31 +1,39 @@
-# .goreleaser.yaml
 builds:
-- skip: true # skip build for now since we have another mechanism to build all kubestellar binaries
-  id: kubestellar-syncer
-  main: ./cmd/syncer
-  binary: kubestellar-syncer
+- id: "controller-manager"
+  main: ./cmd/controller-manager
+  binary: bin/controller-manager
+  ldflags:
+  - "{{ .Env.LDFLAGS }}"
   goos:
   - linux
-  - darwin
   goarch:
   - amd64
   - arm64
-  - s390x
+  ignore:
```

Comment:
Why do we need to "ignore" something not included in the outer product above?

Reply:
Good catch, fixed.